core/src/gl/dynamicQuadMesh.cpp

```diff
@@ -1,41 +1,28 @@
 #include "gl/dynamicQuadMesh.h"
+#include "gl/error.h"
 #include "gl/renderState.h"
 #include "gl/shaderProgram.h"
 
 namespace Tangram {
 
 GLuint QuadIndices::quadIndexBuffer = 0;
 int QuadIndices::quadGeneration = -1;
-std::atomic<int> QuadIndices::meshCounter(0);
```

Comment:
What was the purpose of `meshCounter` here?
I see it was used with every construction of `DyanmicQuadMesh`.

This could be the issue resulting in quad mesh issues with android gl context losses!